tabbycat/results/templates/ballot/ballot_set.html

```diff
@@ -1,5 +1,5 @@
-{% if pref.teams_in_debate == 'bp' and debate.round.is_break_round %}
-  {% include "ballot/bp_elimination_ballot.html" %}
+{% if pref.elim_ballot_nom and debate.round.is_break_round %}
+  {% include "ballot/elimination_ballot.html" %}
```

Comment:
What happens if `pref.teams_in_debate == 'bp'` but `pref.elim_ballot_nom == False`? Does it show the standard BP ballot?